Tooling/ScriptGenerator/AppInputs.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <copyright file="AppInputs.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using CommandLine;
+
+namespace ScriptGenerator;
+
+/// <summary>
+/// The console application argument inputs.
+/// </summary>
+public class AppInputs
+{
+    /// <summary>
+    /// Gets or sets full file path to the source file for processing.
```

Comment:
reword.  also, should "path" be singular or plural?
```suggestion
    /// Gets or sets the full file path to the source file for processing.
```
OR...
```suggestion
    /// Gets or sets a full file path to the source file for processing.
```

Reply:
It should be singular.

Reply:
Ok, so did you change it to "the full file path' or "a full file path?

Reply:
I changed it to **the** full file path.  
??  Did you not see the change?  If you need to see the change then you have to open it in vscode to verify that I changed it.